src/syntax-tree/module.ts

```diff
@@ -635,32 +633,24 @@ export class Module {
     tryInsert(insertInto: CodeConstruct, insert: CodeConstruct): InsertionType {
         const context = this.focus.getContext();
 
+        // TODO: Workaround for now to test ExprDotMethodStmt statements
+        const focusedNode = insertInto ?? this.focus.onEmptyLine() ? context.lineStatement : context.token;
+
+        if (insert instanceof ExprDotMethodStmt && insertInto === focusedNode) {
+            if (insert.validateContext(this.validator, context) == InsertionType.Valid) {
```

Comment:
Can `insert.validateContext(this.validator, context)` return `InsertionType.DraftMode`?

Reply:
That was the goal, right?

Currently, there is only one code construct that does that: variable references -> either on the beginning of a line, or within an expression.

Reply:
Ok, just making sure because you only had a case for if it was valid here.